src/components/GridTemplates/DetailsPage1/_index.scss

```diff
@@ -0,0 +1,35 @@
+////
+/// Details page template 1 styles.
+/// @group accordion
+/// @copyright IBM Security 2018
+////
+
+@import '../../../globals/namespace/index';
+@import '../../../globals/layout/index';
+
+@include export-namespace($name: 'details-page-1') {
+  // Note that this column spans 6:
+  .#{$namespace}details-page-1__section {
+    display: flex;
+
+    // Imitating styles for a column that spans 2 (out of 6)
+    // & also has typical padding:
+    .#{$namespace}details-page-1__section__term {
+      flex: 0 0 100%;
```

Comment:
👏 I remember you mentioning we could pull this out into a helper or `mixin` to determine and provided these values - have you given this further thought?

Reply:
I think that effort should be separate. It would require some testing across different layouts.

Reply:
Fair point!